classes/core/APIRouter.inc.php

```diff
@@ -77,7 +80,16 @@ function route($request) {
 		// Ensure slim library is available
 		require_once('lib/pkp/lib/vendor/autoload.php');
 
-		$sourceFile = sprintf('api/%s/%s/index.php', $this->getVersion(), $this->getEntity());
+		$sourceFile = null;
+		$entity = $this->getEntity();
+		$sourceFile = sprintf('api/%s/%s/index.php', $this->getVersion(), $entity[0]);
+
+		if (!empty($entity[1])) {
```

Comment:
Entities handlers URI prefix is `/{contextPath}/api/{version}/{handlerPath}/`. Then depending of the endpoint, additional URI parts will be appended. With @NateWr we had decided to split backend and public endpoints. Initially I suggested to have all backend endpoints under `/backend`. The downside with that approach considering the initial implementation, as Nate noted, is that all backend endpoints would be defined under the same handler, which is not so convenient and I agree with Nate. So the idea behind this fix is to have a way to split backend endpoints into multiple handlers.